Makefile.am

```diff
@@ -61,44 +61,25 @@ pylint:
 
 if TESTS_ENABLED
 test: all pylint
-	@rm -f tests/error_occurred;
-	@rm -f $(TEST_SUITE_LOG)
-	@touch $(TEST_SUITE_LOG)
-	for plugin in ${PLUGINS} ${LIBRARY_TESTS} ${UTILS_TESTS}; do \
-		$(MAKE) test-plugin-$$plugin || touch tests/error_occurred; \
-	done
-	$(MAKE) test-lvm-dbus || touch tests/error_occurred
-	@test ! -f tests/error_occurred
+	sudo $(TEST_PYTHON) tests/run_tests.py --lucky
```

Comment:
``--lucky`` should only be used in ``test-all``. Or at least that's how it was supposed to work before.